src/olympia/addons/tasks.py

```diff
@@ -417,3 +426,42 @@ def disable_addons(addon_ids, **kw):
     addons.update(status=amo.STATUS_DISABLED, _current_version=None)
     File.objects.filter(version__addon__in=addons).update(status=amo.STATUS_DISABLED)
     index_addons.delay(addon_ids)
+
+
+@task
+@use_primary_db
+def flag_high_hotness_according_to_review_tier():
+    from olympia.reviewers.models import UsageTier
+
+    due_date_generator = get_staggered_review_due_date_generator()
+    usage_tiers = UsageTier.objects.filter(
+        # We don't compute hotness below that, other signals could still apply.
+        lower_adu_threshold__gte=MINIMUM_ADU_FOR_HOTNESS_NONTHEME,
```

Comment:
We'll want them to exist for other purposes - there will be other signals to track (there is no issue for it yet, but one signal is around abuse reports)

Reply:
What would an abuse report UsageTier instance look like, if it doesn't have any adu thresholds?  Is there a concept a UsageTier "type" that hasn't been implemented in the model yet?

Reply:
And, I would assume, at the point there is a "type" we'd just filter on that instead?

Reply:
As above: we'll want these tiers to exist in the database for other purposes - there will be other signals to track (there is no issue for it yet, but one signal is around abuse reports) and just for consistency's sake it's probably a good thing to have all the tiers defined in the database.

Reply:
I'd add a `abuse_report_something_something_threshold` field to the model. The tiers would remain the same, as they are defined independently of the signals we care about.

Reply:
if we're going to be using these tiers generically for things other than add-on usage (adu) then maybe a type (`addon_adu`?) and a generic `lower_threshold`, etc field would be more scalable?

Reply:
The tiers are all going to be defined with add-on usage, but the conditions for signals won't necessarily all be defined for each tier. Specifically for hotness, we don't care about the lower usage tiers because we know their hotness is not computed.